### PR TITLE
Speccpu no disk

### PR DESCRIPTION
**Pull request by dvalinrh**

### CODE REVIEW

**Comment by frival** [comment]
This isn't what I had envisioned.  What I was hoping for was logic that would find the physical filesystem with the most free space (usually /home but not always) and then install there.  If we're going to use a pre-installed kit we also need to consider making the run do a 'runcpu --action=clobber' first just to be sure we rebuild; we could have a --installed combined with a --no-clobber if we want it to skip the clobber step.

File: speccpu2017/run_speccpu [hunk 1]
```diff
@@ -28,6 +28,15 @@ arguments="$@"
...
 disks_found=0
+installed=0
+speccpu_run_dir=""
+speccpu_run="/speccpu_run"
+speccpu_iso_mnt="/speccpu"
+speccpu_iso_file=""
+uploads=""
+give_usage=0
+use_mfs=""
+run_loca_selected=0
```
Comment (frival) [inline speccpu2017/run_speccpu:39]: Suggestion s/run_loca_/run_loc_/ - loc is the more common shorthand for location.

File: speccpu2017/run_speccpu [hunk 2]
```diff
@@ -28,6 +28,15 @@ arguments="$@"
...
 gen_args_back="$@"
 disks_found=0
+installed=0
+speccpu_run_dir=""
+speccpu_run="/speccpu_run"
+speccpu_iso_mnt="/speccpu"
+speccpu_iso_file=""
+uploads=""
+give_usage=0
+use_mfs=""
```
Comment (frival) [inline speccpu2017/run_speccpu:38]: For the option let's call it "no_disk" which we document as "use the existing filesystem with the most amount of free space".

File: speccpu2017/run_speccpu [hunk 3]
```diff
@@ -267,26 +294,82 @@ while [[ $# -gt 0 ]]; do
...
+			speccpu_run_dir=${2}
+			shift 2
+		;;
+		--use_mfs)
+			if [ $run_loca_selected -ne 0 ]; then
+				exit_out "Error: --use_mfs run location already designated, cannot designate multiple"
+			fi
+			run_loca_selected=1
+			pbench_arg_list="${pbench_arg_list} $1"
+			use_mfs=`df --output=avail,target,source | grep -v Mounted | grep -v tmpfs | sort -n | tail -1 | xargs | sed "s/ /:/g" | cut -d: -f2`
```
Comment (frival) [inline speccpu2017/run_speccpu:347]: And then here we can just call it "use_fs" since it doesn't have to mirror the now-replaced use_mfs.

**Review by frival — CHANGES_REQUESTED** [review]
Suggested a couple naming changes.  Not mandatory but LMK what you think.

**Comment by dvalinrh** [comment]
Making naming changes.

File: speccpu2017/run_speccpu (hunk 1 above)
Comment (dvalinrh) [reply speccpu2017/run_speccpu:39]: Done.

**Review by frival — APPROVED** [review]
Assuming both paths have been tested, LGTM.

**Pull request merged**